concordium-sdk/src/main/java/com/concordium/sdk/responses/poolstatus/BakerPoolStatus.java

```diff
@@ -15,7 +15,7 @@
 @Jacksonized
 @Builder
 @EqualsAndHashCode(callSuper = false)
-public class BakerPoolStatus extends PoolStatus {
+public class BakerPoolStatus {
 
     private final PoolType poolType = PoolType.BAKER_POOL;
```

Comment:
Maybe PoolType can be entirely removed? It would remove a getter for PoolType in this class, but i don't see why anyone using a clientV2 would use this getter, as it is always `BakerPoolStatus` being returned by the client.